cli/azd/pkg/contracts/envelope.go

```diff
@@ -10,6 +10,7 @@ type EventDataType string
 const (
 	ConsoleMessageEventDataType EventDataType = "consoleMessage"
 	OperationStart              EventDataType = "operationStart"
+	Endpoint                    EventDataType = "endpoint"
```

Comment:
I wonder if we should consider something like a `serviceCreated` object with `endpoint` and all other useful information.

Reply:
It can be changed in the future, for sure.
Right now is just asking the minimum info required to implement the expected behavior.